app/controllers/build_estimates_controller.rb

```diff
@@ -2,10 +2,11 @@ class BuildEstimatesController < ApplicationController
   include Wicked::Wizard
   include StepsHelper
 
-  steps(*STEPS_WITH_PROPERTY)
+  steps(*StepListerService.call.map(&:name))
```

Comment:
Would it be possible to separate out the feature from the steps refactor? I'm worried that this is 'too much too soon' (although I agree with the basic premise, that the step switching logic is slowly getting out of hand) but think we may want a 'more radical' solution that can't be defined by YAML

Reply:
You're right, it's too much of a refactor to bung in here. I'll remove it for now.